src/components/add-transaction-dialog/add-transaction-dialog.js

```diff
@@ -1,28 +1,69 @@
 import template from "./add-transaction-dialog.html";
+
 import { MDCDialog } from "@material/dialog";
 import { MDCSelect } from "@material/select";
 import { MDCTextField } from "@material/textfield";
 import { MDCRadio } from "@material/radio";
 
+import { CustomSelectComponent } from "../custom-select/custom-select";
+
+import { AccountListService } from "../../services/account-service";
+import { TagListService } from "../../services/tag-service";
+
 export class AddTransactionComponent {
   constructor(mountPoint, props) {
     this.mountPoint = mountPoint;
     this.props = props;
   }
 
+  getStoredAccounts() {
+    AccountListService.get()
+      .then(accounts => this.showAccounts(accounts))
+      .catch(e => console.error(`get accounts: ${e.message}`));
+  }
+
+  showAccounts(accounts) {
```

Comment:
you can use es6 default parameters here like this 

```js
showAccounts(accounts = []) {... }
```
this way you won't have to perform 2 checks  for the case if accounts is not present